### ISSUE TITLE
Add `thisArg` to the functions

### ISSUE THREAD

**Issue by brianmhunt** [One-com/knockout-transformations#7]
It would save on some more complex scenarios to have the functions take a `thisArg`, much like the core e.g. `subscribe` and `computed` functions.

Cheers

**Comment by sunesimonsen** [comment]
Makes sense, I'll see when I have some time for it. PR's are of cause welcome ;-)
